### PR TITLE
Managed Network Resource Provider OpenAPI Specs

### PR DESCRIPTION
**Pull request by yanfa317**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by yanfa317** [comment]
Hi,

We are trying to create swagger for our service ManagedNetworkController, and it is our repo to do it. We are waiting for approver to approve it.

Thanks,
Fan

File: specification/managednetwork/resource-manager/readme.go.md
Comment (ArcturusZhang) [inline specification/managednetwork/resource-manager/readme.go.md]: Lack of another ``` symbol that encloses the yaml code block here

**Comment by michaeljqzq** [comment]
Link to the one in private repo https://github.com/Azure/azure-rest-api-specs-pr/pull/814

**Comment by NullMDR** [comment]
@openapibot sdkautomation rebuild

**Review by michaeljqzq — APPROVED** [review]

**Pull request merged**
